WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -1,8 +1,13 @@
 import SwiftUI
+import class WooFoundation.CurrencyFormatter
+import class WooFoundation.CurrencySettings
 
 final class PointOfSaleDashboardViewModel: ObservableObject {
-    @Published var products: [POSProduct]
-    @Published var productsInCart: [CartProduct] = []
+    @Published private(set) var products: [POSProduct]
+    @Published private(set) var productsInCart: [CartProduct] = []
+    @Published private(set) var formattedCartTotalPrice: String?
```

Comment:
Should we use non-optional for `formattedCartTotalPrice`? The current data type infers that a formatted price might not exist, while the actual output should be that it's either formatted or couldn't be formatted, but some sort of fallback string would be expected? (in this case the "-" used as a nil-coalescing fallback directly in the view)

Reply:
The prices can also be nil before the order is synced, if we decide to use the order's items subtotal instead of a local sum. We'll also show a placeholder UI later. @bozidarsevo feel free to update this to a non-optional with a default string when you take over the order total.